semcore/icon/src/Icon.jsx

```diff
@@ -35,11 +35,23 @@ function Icon(props, ref) {
     keyboardFocused: keyboardFocused,
   });
 
+  function onKeyDown(event) {
+    if (props.onKeyDown) {
+      props.onKeyDown();
```

Comment:
Do not recall default handlers directly, they are already implicitly binded by babel plugins (in short: remove this line)